### PR TITLE
Add validator for Epoch with no scheme

### PR DESCRIPTION
**Pull request by toddburnside**

### CODE REVIEW

File: modules/core/shared/src/main/scala/lucuma/core/math/validation/MathValidators.scala [hunk 1]
```diff
@@ -17,6 +17,9 @@ trait MathValidators {
   val epoch: InputValidSplitEpi[Epoch] =
     InputValidSplitEpi.fromPrism(Epoch.fromString, "Invalid epoch")
 
+    val epochNoScheme:InputValidWedge[Epoch] =
```
Comment (cquiroz) [inline modules/core/shared/src/main/scala/lucuma/core/math/validation/MathValidators.scala:20]: Is it miss indented?

Comment (toddburnside) [reply modules/core/shared/src/main/scala/lucuma/core/math/validation/MathValidators.scala:20]: Oh, probably. I need to re-enable scalaFmt for me locally.

**Review by cquiroz — APPROVED** [review]
LGTM

**Review by rpiaggio — COMMENTED** [review]
LGTM

**Pull request merged**